dhall/tests/format/escapeSingleQuotedOpenInterpolationB.dhall

```diff
@@ -1,3 +1 @@
-''
-''${
-''
\ No newline at end of file
+"\${\n"
```

Comment:
Are these missing newlines expected? Do they matter at all?

Reply:
Yeah, they are expected.  The test suite requires that the file exactly matches what the pretty-printing logic produces (including the lack of a trailing newline)